src/components/Footer/Footer.js

```diff
@@ -66,20 +69,28 @@ const Footer = () => (
 
         <ul className={styles['footer__list']}>
           <li className={styles['footer__list__first__row']}>
-            <a target='blank' className={styles['footer__list__link']} href='#'>
-              <img src='https://www.clipartmax.com/png/middle/35-351662_update-to-our-address-email-icon-png.png'></img>
+            <Link to='/#' className={styles['footer__list__link']}>
+              <Icon
+                className={styles['footer__list__icon']}
+                name='email'
+                icon={getIcon('email')}
+              />
               contact@hoffmann.ai
-            </a>
+            </Link>
           </li>
           <li className={styles['footer__list__first__row']}>
-            <a target='blank' className={styles['footer__list__link']} href='#'>
-              <img src='https://venus-sauna.fr/wp-content/uploads/2019/12/phone.png'></img>
-              06 21 19 56 74
-            </a>
```

Comment:
Same here, the previous tag should be kept. There is a feature in `a` tags that allows you to setup a phone call by clicking on the number. That's what we want here.